cloud/configs/nginx.conf

```diff
@@ -5,6 +5,10 @@
 # LICENSE file in the root directory of this source tree. An additional grant
 # of patent rights can be found in the PATENTS file in the same directory.
 
+# WARNING: Changes to this file will not be reflected until the
+# running nginx instance is explicity reloaded `nginx -s reload` or
```

Comment:
Looking around I found out that reloading a config is a common webserver problem, and nginx actually supports an in-place config update signal that does not require a restart and does not interrupt any connections. If we think it's a good idea I can open another pull request to do this in-place reload with the fab deploy, but didn't want to hold this PR up with mixed changes.